web/api/v1/api.go

```diff
@@ -1868,3 +1894,20 @@ OUTER:
 	}
 	return matcherSets, nil
 }
+
+func parseLimitParam(limitStr string) (limit int, err error) {
+	limit = math.MaxInt
+	if limitStr != "" {
```

Comment:
Probably better to return early:
```go
if limitStr == "" {
  return limit, nil
}
```

Reply:
Done.

Reply:
So now you don't need to check `!= ""`.  This is a Go style point: the code moves further to the left (is less indented) so is a little easier to read.